src/pages/posts/[slug].js

```diff
@@ -98,10 +124,23 @@ export async function getStaticProps({ params = {} } = {}) {
 
   const socialImage = `/images/${params?.slug}.png`;
 
+  const { categories, postId } = post;
+  const category = categories.length && categories[0];
+  let { name, slug } = category;
+  name = undefined;
```

Comment:
won't this and slug override any value that  comes in?

Reply:
oopss sorry about that, testing undefined values 😅